Containers/nextcloud/config/smtp.config.php

```diff
@@ -0,0 +1,20 @@
+<?php
+if (getenv('SMTP_HOST') && getenv('MAIL_FROM_ADDRESS') && getenv('MAIL_DOMAIN')) {
+  $CONFIG = array (
+    'mail_smtpmode' => 'smtp',
```

Comment:
I'm wondering if this could not already work out of the box with https://github.com/nextcloud/server/pull/3966

Reply:
Just looking at the code and assuming that the kubernetes environment variables are properly passed to the PHP process it could already work with `NC_mail_smtpmode=smtp`

Reply:
Yes possibly but I find it cleaner via config.php. also cia direct env is rather limited in what you can do

Reply:
Btw, I copied this file from https://github.com/nextcloud/docker/blob/master/.config/smtp.config.php